### ISSUE TITLE
How can I add custom snippets to postman for other users

### ISSUE THREAD

**Issue by krogold** [postmanlabs/newman#1106]
1. Newman Version (can be found via `newman -v`):
Postman v5.0.2
 2. OS details (type, version, and architecture):
Windows 10

Hello, 
I'm currently using postman to automate some tests and everything I'm doing is a preparatory work for other people. They may not be developpers at all and may have difficulties to handle javascript testing.
I would like to know if it is possible for me to write code snippets, like the ones available in postman, an to import them in postman so these users can use them to ease test development ?

thank you

Alexandre

**Comment by kunagpal** [comment]
@krogold Are you referring to the ability to add custom scripts? If so, please see https://github.com/postmanlabs/postman-app-support/issues/675.

**Comment by krogold** [comment]
Exactly. If found this feature very useful as, when I started using postman, I was a total beginner in javascript. As some of my teammates are not even developpers, I would be happy to provide them with snippets that they would just assemble together, like a puzzle, after functional features only so they would just have to do minor customization. It would be awesome to have it directly in postman interface. 
What I intend to do for now is to build them outside postman and the other users will copy/paste them into postman ...

**Comment by kunagpal** [comment]
Related to #734

**Comment by efrenette11** [comment]
+1

**Comment by pinalbhatt** [comment]
+1
